include/pycppad/codegen/cppadcg-scalar.hpp

```diff
@@ -35,13 +34,12 @@ namespace pycppad
 
       CGVisitor<Scalar>::expose();
 
-
       pycppad::ADVisitor<CGScalar>::expose("ADCG");
       pycppad::ADFunVisitor<CGScalar>::expose("ADCGFun");
-      pycppad::IndependentVisitor<VectorADCG>::expose("CGIndependent");
-      pycppad::IndependentVisitor<RowVectorADCG>::expose("CGIndependent");
+      pycppad::IndependentVisitor<VectorADCG>::expose("Independent");
```

Comment:
Class `Independent` expects vectors of type double. `CGIndependent` accepts vectors of type CG

Reply:
Yes, but I do think having a single naming for it will enhance the usage. In C++, there is only one single function name for various signatures. We should stick to this rule.

Reply:
By the way, Independent is not a class.

Reply:
yes, I see that. Thanks for the overload. It might be worthwhile to do the same for ADCG class as well, what do you think?

Reply:
Can you handle it?

Reply:
I don't see any other way than changing AD to a function instead of a class. It could then return ADD or ADCG type variables. So we would have the API where `AD(CG(10))` returns a type of ADCG(10), and `AD(10)` returns `ADD(10)`. It doesn't look clean though.

Reply:
Yes, we cannot do overloading of class. Except if we introduce the scope.

Reply:
I don't like the idea of user not knowing which type would be returned by AD. It would be better to leave the current implementation in that case, it is clearer in API in my opinion.

Reply:
If you do `type(obj)` you will get the real type.

Reply:
But this is fine for me with the current status